examples/module-lifecycle/module_lifecycle.bal

```diff
@@ -1,11 +1,11 @@
 import ballerina/io;
 
 // Usually it is an error to import a module without using it, but if you want to import a module because of what its initialization does,
```

Comment:
```suggestion
// Usually, it is an error to import a module without using it but if you want to import a module because of what its initialization does,
```